main.tf

```diff
@@ -1,3 +1,7 @@
+locals {
+  effective_policy_arns = length(var.s3_policy_arns) > 0 ? var.s3_policy_arns : var.additional_policy_arns
```

Comment:
This deprecation is the opposite of how we've done it in the other modules:
https://github.com/Datatamer/terraform-aws-emr/blob/master/main.tf#L3

This prefers the original over the new, where for other modules we've used the new variable over the deprecated variable.

Let's be consistent with the deprecation behavior and use the new variable if it's defined.

Reply:
You're right. We are trying to be consistent thanks for the review I corrected it.